test/tax-lot/tax-lot.e2e-spec.ts

```diff
@@ -0,0 +1,92 @@
+import * as request from "supertest";
+import { INestApplication } from "@nestjs/common";
+import { Test } from "@nestjs/testing";
+import { TaxLotModule } from "src/tax-lot/tax-lot.module";
+import { TaxLotRepository } from "src/tax-lot/tax-lot.repository";
+import { getTaxLotGeoJsonByBblQueryResponseSchema } from "src/gen";
+import { TaxLotRepositoryMock } from "./tax-lot.repository.mock";
+import {
+  DataRetrievalException,
+  InvalidRequestParameterException,
+} from "src/exception";
+import { HttpName } from "src/filter";
+
+describe("TaxLots", () => {
+  let app: INestApplication;
+
+  const taxLotRepository = new TaxLotRepositoryMock();
+
+  beforeAll(async () => {
+    const moduleRef = await Test.createTestingModule({
+      imports: [TaxLotModule],
+    })
+      .overrideProvider(TaxLotRepository)
+      .useValue(taxLotRepository)
+      .compile();
+    app = moduleRef.createNestApplication();
+    await app.init();
+  });
+
+  describe("getTaxLotGeoJsonByBbl", () => {
+    it("should 200 and return documented schema when finding by valid bbl", async () => {
```

Comment:
Place each scenario within its own test. Describe the test in the format 'it("should [expected result] when [scenario happens]")'